.github/workflows/build_wheels.yml

```diff
@@ -52,7 +52,7 @@ jobs:
     if: github.event_name == 'release' && github.event.action == 'published'
     needs: [build_wheels, build_sdist]
     runs-on: ubuntu-latest
-    environment: 
```

Comment:
Should we add the trailing whitespace fixer pre-commit-hook so that these don't reappear?